README.md

```diff
@@ -25,141 +27,106 @@ a recommended tool for more full-featured analysis of JFR files beyond what
-listing available target JVMs. If `CRYOSTAT_AUTH_MANAGER` is not specified
-then a default auth manager will also be selected corresponding to the platform,
-whether that platform is specified by the user or automatically detected. The
-value of this variable should be set to the fully-qualified name of the
-platform detection strategy implementation to use, ex.
-`io.cryostat.platform.internal.KubeEnvPlatformStrategy`.
-
-The environment variable `CRYOSTAT_REPORT_GENERATION_MAX_HEAP` is used to
-configure the maximum heap size used by the container subprocess which forks to
-perform automated rules analysis report generation. The default is `200`,
-representing a `200MiB` maximum heap size. Too small of a heap size will lead
-to report generation failing due to Out-Of-Memory errors. Too large of a heap
-size may lead to the subprocess being forcibly killed and the parent process
-failing to detect the reason for the failure, leading to inaccurate failure
-error messages and API responses.
-
-The environment variables `CRYOSTAT_TARGET_CACHE_MAX_CONNECTIONS` and
-`CRYOSTAT_TARGET_CACHE_TTL` are used to control the target JMX connection cache
-behaviour. `CRYOSTAT_TARGET_CACHE_MAX_CONNECTIONS` specifies how many
-connections may be held open at once. `-1` may be used to leave the cache size
-unlimited.  `CRYOSTAT_TARGET_CACHE_TTL` specifies how long (in seconds) these
-connections will be cached before they are closed due to inactivity.
-
-For logging, Cryostat uses SLF4J with the java.util.logging binding.
-The default configuration can be overridden by mounting the desired
-configuration file in the container, and setting the environment variable
-`CRYOSTAT_JUL_CONFIG` to the path of that file.
-
-Some of Cryostat's dependencies also use java.util.logging for their logging.
-Cryostat disables
-[some of these](https://github.com/cryostatio/cryostat-core/tree/main/src/main/resources/config/logging.properties)
-by default, because they generate unnecessary logs.
-However, they can be reenabled by overriding the default configuration file
-and setting the disabled loggers to the desired level.
+
+### Development Run Outside of Containers
+* `MAVEN_OPTS="-Dcom.sun.management.jmxremote.port=9091 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.autodiscovery=true" mvn clean prepare-package exec:java`.
+
+### Run on Kubernetes/Openshift
+* See the [cryostat-operator](https://github.com/cryostatio/cryostat-operator)
+
+### Run on local podman
+* `run.sh`
+
+### Run on local podman with Grafana, jfr-datasource and demo application
+* `smoketest.sh`
+
+## CONFIGURATION
+
+Cryostat can be configured via the following environment variables
+
+#### Configuration for cryostat-web
+
+* `CRYOSTAT_WEB_HOST`: the hostname used by cryostat-web
+* `CRYOSTAT_WEB_PORT`: the internal port used by cryostat-web
+* `CRYOSTAT_EXT_WEB_PORT`: the external port used by cryostat-web
```

Comment:
The three env vars above aren't really just for -web, they are for the Cryostat backend internal HTTP webserver. `WEB_HOST` and `WEB_PORT` define the address that the webserver binds on, for example.

Reply:
Hm; I'm a little confused here. Should they be `CRYOSTAT_HOST` then? Or I suppose WEB refers to web server rather than cryostat-web?

Reply:
Yea, it was meant to refer to the webserver. It was named this way originally as a contrast to the old `NET` variables which did the same thing but for the raw TCP socket connection.